pages/content/amp-dev/documentation/guides-and-tutorials/contribute/translations@ko.md

```diff
@@ -0,0 +1,82 @@
+---
+$title: 번역 추가하기
+$order: 2
+description: "AMP의 사명은 웹을 모든 사람이 환영할 수 있고 접근하기 쉬운 장소로 만드는 것을 포함하며, 현지화된 문서 없이는 이 목표를 달성할 수 없습니다. AMP 문서를 번역하는 것은 AMP 커뮤니티의 성공과 건강을 돕는 필수적인 과정입니다."
+
+formats:
+  - websites
+  - stories
+  - ads
+  - email
+---
```

Comment:
Thanks for the translation! One small nit - the only thing that should be in the front matter is the title. Could you please remove everything else?

Reply:
I fixed it. And improved other page(index - How to contribute), too.